src/market.rs

```diff
@@ -286,6 +289,26 @@ impl GenoaMarket {
         })
     }
 
+    pub fn update_news(&mut self) {
+        // Updating news by introducing noise and maintaining the value 
+        // between -1 and 1 (using the Sigmoid function).
+        self.news_indicator =
+            1. / (1. + (-self.news_indicator + rand::thread_rng().gen_range(-2.5..2.5)).exp());
+    }
+
+    pub fn get_news(&self) -> f32 {
+        // A simple value that indicates the status of a market.
```

Comment:
If you putt the comment before the function with three slashes, it will show up in documentation:
```
/// This function busts balls
fn ball_buster() {...}
```